charts/eks-anywhere-packages/values.yaml

```diff
@@ -85,4 +85,14 @@ controller:
       memory: 50Mi
     limits:
       cpu: 500m
-      memory: 300Mi
\ No newline at end of file
+      memory: 300Mi
+# Additional Variables to configure cronjob
+cronjob:
+  name: ecr-refresher
+  # -- ECR refresher repository name.
+  repository: "eks-anywhere-packages"
```

Comment:
```suggestion
  repository: "packages-token-refresher"
```

Reply:
Do we need build tooling to also push this to a seperate repository?

Reply:
yes and cdk to create the repo, Jonah to make sure it gets promoted.

Reply:
This should be:
`repository: "eks-anywhere-packages/ecrtokenrefresher"` per [build tooling PR](https://github.com/aws/eks-anywhere-build-tooling/pull/1107). I can re-name it to something else, but I was hoping to use this structure to be clear that this image is built from the `eks-anywhere-packages` project.

Reply:
Ok. Then the update here will be:
`repository: "eks-anywhere-packages/ecr-token-refresher"`
I will update the build tooling PR.

Reply:
Seems really odd to me to create a repository that has the path of another repository in it. I imagine it will work though.

Reply:
Changed name to match

Reply:
I was trying to follow the pattern of `org/project`, which was used by many projects in the build-tooling. But based on today's conversation, I don't think we want to move forward with this naming convention anyway (though we still don't have a conclusion on this yet). Given this is a short-term solution, I'm thinking we can update it to something like `repository: "ecr-token-refresher"` or something similar (whatever makes sense logically).

Just let me know what you guys decide on, and I will update build-tooling accordingly.